qtawesome/iconic_font.py

```diff
@@ -616,6 +616,24 @@ def set_custom_icon(self, name, painter):
         """
         self.painters[name] = painter
 
+    def install_fonts_system_wide(self):
+        """Install bundled fonts system wide on Windows."""
+        if os.name == "nt":
+            if windll.shell32.IsUserAnAdmin():
+                # Execute logic to install bundled fonts system wide
+                fonts_directory = os.path.join(
+                    os.path.dirname(os.path.realpath(__file__)), "fonts"
+                )
+                self._install_fonts(fonts_directory, system_wide=True)
+            else:
+                # Call `qta-fonts` using `runas` to prompt user for admin elevation
+                warnings.warn(
+                    "Need for admin privileges to install bundled fonts. "
+                    "A prompt to get admin privileges and relaunch the command "
+                    "will be shown..."
```

Comment:
```suggestion
                    "Admin privileges are needed to install the bundled fonts. "
                    "A prompt to get them will be shown and after that this "
                    "command will be relaunched..."
```

Improve wording for this message.